### PR TITLE
s-t integrations: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

s-t integrations: 
- apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sensibo.markdown [hunk 1]
```diff
@@ -166,7 +166,7 @@ For devices which are also controlled in other ways or often goes out of sync wi
 
 With this service you can tell Sensibo if your device is currently running or not without sending a new command to you device.
 
-### Pure Boost
+### Pure boost
```
Comment (joostlek) [inline source/_integrations/sensibo.markdown]: Isn't this a brand thing?

File: source/_integrations/sensibo.markdown [hunk 2]
```diff
@@ -178,7 +178,7 @@ Using Geo integration for Pure Boost is only possible by pre-configuration of Pr
 
 You can enable a timer with a custom delay using the service `sensibo.enable_timer` that is provided.
 
-### Climate React
+### Climate react
```
Comment (joostlek) [inline source/_integrations/sensibo.markdown]: Same

File: source/_integrations/sensibo.markdown (hunk 1 above)
Comment (c0ffeeca7) [inline source/_integrations/sensibo.markdown]:
```suggestion
### Pure Boost
```

File: source/_integrations/sensibo.markdown (hunk 2 above)
Comment (c0ffeeca7) [inline source/_integrations/sensibo.markdown]:
```suggestion
### Climate React
```

**Review by joostlek — APPROVED** [review]

File: source/_integrations/sensibo.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_integrations/sensibo.markdown]: Thank you for the catch, @joostlek

**Pull request merged**